docs/configuration/index.md

```diff
@@ -449,11 +449,11 @@ this message is used as an extension of the comment added by default by Packit.
 
 Use this key to be notified about errors of the downstream jobs (Koji build, Bodhi update, pull from upstream).
 The value can be a URL of any GitHub/GitLab/Pagure project including dist-git
-where issues are enabled and Packit has an identity on that git forger instance.
+where issues are enabled and Packit has an identity on that git forge instance.
 Alternatively, you can set up a dedicated project for receiving such notifications.
 (Let us know if you need another instance to be supported.)
-It does not need to be an upstream repository,
-you can also enable an issue tracker for the dist-git repository
+It does not need to be an upstream repository (and in case you do not maintain it, it also **should not** be).
+You can also enable an issue tracker for the dist-git repository
```

Comment:
Didn't we find out that this is in fact not possible? :sweat_smile:

Reply:
I was not 100% sure if I remembered that correctly :D whether it wasn't impossible only via UI..I will remove it then